fashion-messaging-app-ui/src/components/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,22 @@
+import Navbar from "../Navbar/Navbar";
+import FashionItems from "../FashionItems/FashionItems";
+import Search from "../Search/Search";
+import "./Homepage.css";
+
+
+function Homepage({loading, photos, handleSearch}) {
```

Comment:
Please ignore the loading prop I added here. I added it because I was about to start working on the loading state feature for my next pull request but I did not see this before pushing

Reply:
> in the future, if you notice something like this after pushing, I think you can split the commit and repush just the commit with the code you want to merge. that will save the work you did for a future push without pushing unnecessary changes to your PR.

Okay, got it!